app/src/main/res/values/strings.xml

```diff
@@ -772,4 +772,8 @@
   <string name="blood_sugar_unit_selection_choose">Choose</string>
   <string name="blood_sugar_unit_selection_done">DONE</string>
 
-</resources>
+  <!-- BP passport sheet -->
+  <string name="sheet_bp_passport_register_new_patient">Register New Patient</string>
+  <string name="sheet_bp_passport_add_to_existing_patient">Add to existing Patient</string>
```

Comment:
Can we check the word casing here, please?  Only the first word should be captalised.